vignettes/datatable-sd-usage.Rmd

```diff
@@ -73,7 +73,13 @@ The first way to impact what `.SD` is is to limit the _columns_ contained in `.S
 Pitching[ , .SD, .SDcols = c('W', 'L', 'G')]
 ```
 
-This is just for illustration and was pretty boring. But even this simply usage lends itself to a wide variety of highly beneficial / ubiquitous data manipulation operations:
+This is just for illustration and was pretty boring. In addition to accepting a character vector, `.SDcols` also accepts:
+
+1. a function such as `is.character`
+2. name searching with `patterns()`
+3. Integer and boolean vectors.
```

Comment:
Integer could be lowercase, and instead of boolean it is better to use "logical", and we don't need dot at the end